DeBugIt/Game.cpp

```diff
@@ -0,0 +1,130 @@
+﻿#include "Game.h"
+#include <SFML/Graphics.hpp>
+#include <SFML/Window.hpp>
+#include <iostream>
+#include <algorithm>
+#include "Monster.h"
+#include "Base.h"
+#include "Wave.h"
+#include "BuildPlace.h"
+#include "Tower.h"
+#include "Menu.h"
+
+
+
```

Comment:
In general: Need to try building more clear architecture before jumping into implementation. Draw class diagrams, sequence diagrams, and think who will be responsible for what. With that in mind, you could eliminate a lot of problems before they're even there.